plugins/org.jboss.tools.openshift.common.core/pom.xml

```diff
@@ -9,4 +9,64 @@
 	<groupId>org.jboss.tools.openshift.plugins</groupId>
 	<artifactId>org.jboss.tools.openshift.common.core</artifactId> 
 	<packaging>eclipse-plugin</packaging>
+	<build>
+		<pluginManagement>
```

Comment:
Why are these m2e settings needed? This is only required if you have maven natures enabled in Eclipse, which brings no benefit imho but rather slows the build.